CHANGELOG.rst

```diff
@@ -5,6 +5,27 @@ Synapse Changelog
 *****************
 
 
+v2.89.0 - 2022-03-31
+====================
+
+Features and Enhancements
+-------------------------
+- Update the ``meta`` model.
+  (`#2621 <https://github.com/vertexproject/synapse/pull/2621>`_)
+
+  ``meta:ruleset``
+    Add a new form to denote the collection of a set of nodes representing
+    rules, which should be linked together with a ``has`` light edge.
+
+- Add additional filter options for the Storm ``merge`` command.
+  (`#2615 <https://github.com/vertexproject/synapse/pull/2615>`_)
+- Update the ``BadSyntaxError`` exception thrown when parsing Storm queries to
+  additionally include additional line and column when available. Fix an issue
```

Comment:
the second "additional" in "additionally include additional" feels a little redundant to me but maybe I'm just being picky

Reply:
"additionally include line and column  when available" seems sufficient